pkg/proxy/ipvs/proxier.go

```diff
@@ -525,6 +528,23 @@ func (handle *LinuxKernelHandler) GetModules() ([]string, error) {
 	return append(mods, bmods...), nil
 }
 
+// GetKernelVersion returns currently running kernel version.
+func (handle *LinuxKernelHandler) GetKernelVersion() (*version.Version, error) {
+	kernelVersionFile := "/proc/sys/kernel/osrelease"
+	out, err := handle.executor.Command("cut", "-f1", "-d", " ", kernelVersionFile).CombinedOutput()
```

Comment:
A bit overdue, but we should use `strings.Fields` here instead of execing into `cut`. Can we fix it here or add a TODO comment?

Reply:
I read file with limiting buffer size as suggested in the other issue you linked. AFAIK we don't need to call `Fields` as this file contains only kernel version in this file. But i used it to match older behavior. I checked [kernel documentation](https://www.kernel.org/doc/Documentation/sysctl/kernel.txt) but couldn't conclude if this file can have multiple columns of text.

Reply:
I also noticed we removed the `strings.TrimSpace` check, I agree it's probably not needed here because of `strings.Fields` but I can't say confidently that it wasn't added for no  reason.